src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineStepVisitor.java

```diff
@@ -95,23 +95,25 @@ public PipelineStepVisitor(WorkflowRun run, @Nullable final FlowNode node) {
     }
   }
 
-  @Override
-  public void parallelBranchStart(
+  //@Override
+  public void xxxparallelBranchStart(
```

Comment:
not used anymore?

Reply:
Sorry, work still needed to get all the use-cases working (I thought I could remove this, but it break other things).